### PR TITLE
PYIC-7346: P1 journeys extended in core-back api-tests

### PR DESCRIPTION
**Pull request by AmritSidhu**
### What changed

- low-confidence evidence_requested validation api-tests
- medium-confidence evidence_requested validation api-tests
- photo/no-photo id low-confidence thin file api-tests
- unsuccessful KBV questions for low confidence users photo/no-photo id api-tests

Subsequent PR: https://github.com/govuk-one-login/ipv-core-tests/pull/881

### Issue tracking

- [PYIC-7346](https://govukverify.atlassian.net/browse/PYIC-7346)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

[PYIC-7346]: https://govukverify.atlassian.net/browse/PYIC-7346?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api-tests/features/p1-journeys.feature [hunk 1]
```diff
@@ -215,3 +215,215 @@ Feature: P1 journey
...
+    When I submit 'kenneth-score-2' details with attributes to the CRI stub
+      | Attribute          | Values                                          |
+      | evidence_requested | {"scoringPolicy":"gpg45","verificationScore":2} |
+    Then I get a 'page-ipv-success' page response
+    When I submit a 'next' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a 'P2' identity
+
+  Scenario: P1 Validate the Evidence requested for medium confidence journey in HMRC KBV
```
Comment (blakeyp) [inline api-tests/features/p1-journeys.feature]: Hmm I'd say we don't really need these `Validate the Evidence requested for medium confidence journey` ones as they're already covered in the P2 web journey scenarios? and soon we'll be enabling P1 by default

File: api-tests/features/p1-no-photo-id.feature [hunk 2]
```diff
@@ -298,3 +298,57 @@ Feature: P1 No Photo Id Journey
     Then I get a 'page-ipv-identity-postoffice-start' page response with context 'lastChoice'
     When I submit a 'end' event
     Then I get a 'pyi-escape' page response
+
+  Scenario: P1 thin file for low confidence users without photo ID
```
Comment (blakeyp) [inline api-tests/features/p1-no-photo-id.feature]: I think this is already covered by `P1 DCMAW after KBV dropout Journey` in p1-journeys.feature ?

Comment (AmritSidhu) [reply api-tests/features/p1-no-photo-id.feature]: The key difference here is that we are specifying activity score 0 whereas that test uses an activity score of 1. however they do give the same result

Comment (blakeyp) [reply api-tests/features/p1-no-photo-id.feature]: Ah ok, but yeah that shouldn't affect these scenarios (activity score isn't required for P1)

**Review by blakeyp — APPROVED** [review]

**Pull request merged**